workers/unity/Assets/Gdk/Core/Error/ForwardingDispatcher.cs

```diff
@@ -0,0 +1,67 @@
+using Improbable.Worker;
```

Comment:
Nit: How about renaming the `Error` folder to something like `Logging`?

Reply:
Yeah thats a good point, I will change it once this gets approved. I don't want to move everything now, it will just confuse everyone when reviewing the PR.